scripts/circle-ci/run-circle-tests.sh

```diff
@@ -7,12 +7,30 @@ function checkDocsBuild {
   make html
 }
 
+# Container 0 - runs tasks not found in the below containers
+
+# Container 1
 CASSANDRA=':atlasdb-cassandra-tests:check'
+
+# Container 2
 SHARED=':atlasdb-tests-shared:check'
 ETE=':atlasdb-ete-tests:check'
 
+#Container 3
+TIMELOCK_ETE=':atlasdb-timelock-ete:check'
+DROPWIZARD=':atlasdb-dropwizard-tests:check'
+LOCK=':lock-impl:check'
+
+EXCLUDE=($CASSANDRA $SHARED $ETE $TIMELOCK_ETE $DROPWIZARD $LOCK)
```

Comment:
Splitting off a new task means I have to add it to two lists: (1) this EXCLUDE list, and (2) the gradlew args list. 

I wonder if it makes sense to instead track 3 lists like NODE_{1..3}_TESTS, then add those 3 to EXCLUDE_ARGS. This way, I only need to add a new task to one list.